pkg/crypto/mpc/wallet.go

```diff
@@ -45,7 +45,6 @@ func GenerateWallet(cb common.MotorCallback, options ...WalletOption) (*Wallet,
 		}(id)
 	}
 	wg.Wait()
-	cb.OnWalletCreated(true)
```

Comment:
What did this CB do?

Reply:
That must have been old, because that method doesn't exist anymore.